lib/datasource/index.ts

```diff
@@ -15,7 +15,14 @@ import { loadModules } from '../util/modules';
 
 export * from './common';
 
-const datasources = loadModules<Datasource>(__dirname);
+// istanbul ignore next
+function validateDatasource(module, name): boolean {
+  if (!module.getPkgReleases) return false;
+  if (module.id !== name) return false;
+  return true;
+}
+
+const datasources = loadModules<Datasource>(__dirname, validateDatasource);
```

Comment:
We should run this at a test for every datasource.

Reply:
It will get run anyway, because we import `datasource/index.ts` at some point.